### ISSUE TITLE
After updating from Github will not upload the image to Media-Manager

### ISSUE THREAD

**Issue by Evgura** [e107inc/e107#3391]
After updating from Github will not load the image. All permissions are set.
![74555](https://user-images.githubusercontent.com/25720089/44311636-4e94d000-a404-11e8-89ee-51716b9ad02d.jpg)

Or so.

![74556](https://user-images.githubusercontent.com/25720089/44311905-e3013180-a408-11e8-9c92-d024be34f820.jpg)

**Comment by CaMer0n** [comment]
@Evgura Could you please send me the image via Gitter. I will need to test with it.  Thank you.

**Comment by Evgura** [comment]
@CaMer0n I do not have such a messenger, I do not use it.

**Comment by CaMer0n** [comment]
@Evgura No messenger needed, just go here and login with your Github ID: http://gitter.im/CaMer0n

**Comment by Deltik** [comment]
@CaMer0n: Haven't set up a reproduction, but the condition getting hit is this one in [`./e107_web/js/plupload/upload.php` on line 195](https://github.com/e107inc/e107/commit/e5bb5297f68e56537c004cdbb48a30892e9f6f4c#diff-ca60332808d25dc1d058be48d2fd06edR195):

```php
        if(e107::getFile()->isClean($filePath) !== true)
        {
                @unlink($filePath);
                die('{"jsonrpc" : "2.0", "error" : {"code": 103, "message": "Bad File Detected."}, "id" : "id"}');
        }
```

**Comment by CaMer0n** [comment]
Thank you @Deltik ! Indeed, perhaps this is the cause in @Evgura 's situation.

**Comment by Evgura** [comment]
File filetypes.xml exists.

**Comment by CaMer0n** [comment]
@Evgura Have you tried other jpg files?

**Comment by Evgura** [comment]
@CaMer0n I make files .jpg (crop, resize) the same program, and before the update to 2.1.9 with Github everything was fine loaded.
Today returned version 2.1.8 and the same photos that you sent, no problem filled in the site.

**Comment by CaMer0n** [comment]
Please send me some of the .jpg files you made.

**Comment by Evgura** [comment]
On Gitter in the archive.

**Comment by tgtje** [comment]
Just updated a quit fresh live site (github sync), i can not see/detect , nor have any issues uploading..

However i noticed when looking on the above screenshots... where do they come from ?
I do not see (in different paths of upload;  or is mine wrong??) any TABS on media upload like above (youtube/video etc..) Sorry

**Comment by Evgura** [comment]
The problem was solved by updating to version 2.1.9 (e107_v2.x_to_2.1.9_upgrade)
